tests/dist/src/test/java/org/hawkular/apm/tests/dist/AnalyticsServiceITest.java

```diff
@@ -1159,7 +1165,7 @@ public void testGetCompletionTimeMultiFragment() throws Exception {
         assertNotNull(stats);
         assertEquals(1, stats.size());
 
-        assertEquals(1750, stats.get(0).getAverage());
+        assertEquals(1749999, stats.get(0).getAverage());
```

Comment:
This still looks wrong - should be 1750000, unless its an accuracy issue in ES?
If you can't see anything obvious, create a jira and assign to me, to check after this has been merged.